src/packages/blits/.storybook/preview.ts

```diff
@@ -0,0 +1,55 @@
+import Blits from '@lightningjs/blits';
+import type { Preview } from '@storybook/html';
+
+const launchApp = ({name, fn, template}) => {
+
+  // app with single component
+  const App = Blits.Application({
+    components: {
+      [name]: fn
+    },
+    template,
+  })
+
+  // observer waits until app element is found before launching blits app
+  const observer = new MutationObserver((mutationList, observer) => {
+    const appElement = document.getElementById('app');
+
+    if (appElement) {
+      observer.disconnect();
+
+      Blits.Launch(App, 'app', {
+        w: 1920,
+        h: 1080,
+        multithreaded: false,
+        debugLevel: 1,
+        inspector: true
+      })
+    }
+  });
+
+  // start observing root until app element is initialised
+  observer.observe(document, {
+    subtree: true,
+    childList: true
+  });
+
+}
+
+const preview: Preview = {
+  parameters: {
+    controls: {
+      matchers: {
+        color: /(background|color)$/i,
+        date: /Date$/i
+      }
+    },
+    backgrounds: {
+      default: 'dark'
+    }
+  },
+  //@ts-ignore
```

Comment:
instead of using ts-ignore, use ts-expect-error - it informs TS that there's an error thats actually ok instead of telling the compiler to ignore the line entirely(it doesn't super matter here, but it's a better practice)

Reply:
Thanks Anthony, I've just pushed a change for that, have removed the other instances of ts-ignore I had in there too